helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1209,7 +1209,10 @@ operator:
         tag: v1.5.0
         env:
           - name: OTEL_DOTNET_AUTO_PLUGINS
-            value: Splunk.OpenTelemetry.AutoInstrumentation.Plugin, Splunk.OpenTelemetry.AutoInstrumentation
+            value: Splunk.OpenTelemetry.AutoInstrumentation.Plugin,Splunk.OpenTelemetry.AutoInstrumentation
+          - name: OTEL_DOTNET_AUTO_RULE_ENGINE_ENABLED
+            # TODO: Decide to remove or keep this env var, might be a temporary work around
```

Comment:
Add the OTL task number.

Reply:
I'm actually able to get around this TODO buy using the `instrumentation.opentelemetry.io/otel-dotnet-auto-runtime: "linux-musl-x64"` with the dotnet-test app. I'm not sure if this fixes the root issue properly, but we have a follow up ticket (OTL-2843) to address this. Removing this section for now.